src/follower.py

```diff
@@ -188,7 +189,7 @@ def main():
     follower = Follower()
     while True:
         follower.follow()
-
+        log.log(follower, 'follower')
```

Comment:
Does this *need* to be a string? Especially if you're passing the *whole* object. I'm not sure about this.

Reply:
Yes because python 2 doesnt support enums and how else would I get access to the data outside of follower.py? It doesnt make any sense to import follower into a log.

Reply:
You can use type or isinstance to solve this.